cypress/page_objects/calculator.js

```diff
@@ -0,0 +1,55 @@
+class Calculator {
+    
+    initialPage = "https://testsheepnz.github.io/BasicCalculator";
+    buildDropDownList = '#selectBuild';
```

Comment:
It would be less cumbersome and more readable to have "buildVersion: () => cy.get('#selectBuild')". What do you think?

Reply:
I do agree with this solution. It would be easier to read and would save up some time as you have to search for the method that uses this value.

Reply:
Just a follow up, the correct syntax would be "buildVersion = () => cy.get('#selectBuild')", as type annotations are only supported in Typescript files. :)